tsconfig.json

```diff
@@ -29,6 +30,7 @@
     "resolveJsonModule": true,
     "esModuleInterop": true
   },
+  "files": ["node_modules/@storybook/react/dist/ts3.9/client/preview/types-6-0.d.ts"],
```

Comment:
Hmm why did we add this line?

Reply:
There are strange things with our TS configuration 😅 I wanted to actually use the `StoryFn` type and this was a patch to make it accessible. I can back it out though and revert the story to just be a function that returns a react node.

Reply:
I really would love to take a look at our TS configs and see if we can improve some of the long standing by low-priority funniness. Def not needed for this patch tho.

Reply:
The TS configuration is fine, honestly. I don't think those are types you should be reaching for. There are better ones correctly exported from storybook react

Reply:
FWIW I was trying to access the `StoryFN` type because [storybook strongly recommends using CSF](https://storybook.js.org/docs/react/api/csf). 

In a comment above I mentioned that previously I was not able to import the expected types from the react package, but that is no longer an issue.